facebook/__init__.py

```diff
@@ -362,6 +364,18 @@ def debug_access_token(self, token, app_id, app_secret):
         }
         return self.request(self.version + "/" + "debug_token", args=args)
 
+    def get_auth_url(self, app_id, canvas_url, perms=None, **kwargs):
+        """
+        Build link to make login though Facebook Open Authorization (OAuth).
```

Comment:
I think this docstring is a bit confusing. Something like "Build a URL to create an OAuth dialog" or just linking to the relevant part of Facebook's documentation would be better.